### PR TITLE
Faq Content Added

### PR DESCRIPTION
**Pull request by sujitmemane**
Content Updation of Website based on previous website and notion

### CODE REVIEW

File: components/faq.tsx [hunk 1]
```diff
@@ -15,45 +15,41 @@ export function Faq() {
...
+            We offer three batches of mentorship. The first batch is perfect for beginners, covering HTML, CSS, and JavaScript fundamentals through exciting projects. The second batch focuses on mastering ReactJS and building real-world projects. The third batch provides guidance on job hunting, application strategies, and securing referrals.
             </AccordionContent>
           </AccordionItem>
           <AccordionItem value="item-3">
-            <AccordionTrigger>Is it animated?</AccordionTrigger>
+            <AccordionTrigger>Not sure which batch to join?</AccordionTrigger>
             <AccordionContent>
-              Yes. It&apos;s animated by default, but you can disable it if you
-              prefer.
+            If you're confident in your HTML, CSS, and JavaScript skills, join batch 2 to learn ReactJS. Otherwise, start with batch 1 to establish a solid foundation before diving into ReactJS. If you already have impressive skills and projects, join batch 3 to focus on job applications. Otherwise, joining batch 2 to build real-world projects would be beneficial.
```
Comment (ManishBisht777) [inline components/faq.tsx:52]: the `'` in `you're` is causing build error

**Review by ManishBisht777 — CHANGES_REQUESTED** [review]
left some comments

File: components/faq.tsx (hunk 1 above)
Comment (sujitmemane) [reply components/faq.tsx:52]: ok Got it

**Review by ManishBisht777 — APPROVED** [review]
LGTM

**Pull request merged**